api-doc/api.raml

```diff
@@ -0,0 +1,21 @@
+#%RAML 1.0
+title: eLife's Libero walking skeleton
+
+description: |
+    Example of a composed Libero API schema.
+
+baseUri: http://localhost:8085
+
+resourceTypes:
+    articles: !include articles.raml
+
+types:
+    ArticleVersionFront:
```

Comment:
This might need to be rethought. In the event we want to run the article store multiple times (once for scholarly articles, once for blog posts), this wouldn't work? (Thought are we going to allow the same service to be run multiple times?)